integration_test/third_party_apps_data/applications/mongodb/debian_ubuntu/install

```diff
@@ -2,17 +2,27 @@ set -e
 
 sudo apt-get update
 sudo apt-get install -y gnupg wget
-wget --no-verbose --output-document=- https://www.mongodb.org/static/pgp/server-5.0.asc | sudo apt-key add -
+wget --no-verbose --output-document=- https://www.mongodb.org/static/pgp/server-6.0.asc | sudo apt-key add -
```

Comment:
FWIW `apt-key add` is deprecated and will be removed in Debian 12. You're supposed to just place the file in `/etc/apt/trusted.gpg.d/` directly instead (i.e. pass `--output-document=/etc/apt/trusted.gpg.d/mongodb.asc` to `wget`).

Reply:
Removed all occurrences of `apt-get add`.